pathspider/cmd/measure.py

```diff
@@ -135,8 +147,16 @@ def _format_action(self, action):
     parser.add_argument('--output', default='/dev/stdout', metavar='OUTPUTFILE',
                         help=("The file to output results data to. "
                               "Defaults to standard output."))
+    parser.add_argument('--autoname', action='store_true',
```

Comment:
The format for generated names should be deterministic if we are to have these at all. The PTO is entirely based around passing things on file descriptors, as is the current PATHspider workflows. There is no documentation in the Sphinx docs for these options.

Reply:
This feature can be removed if you want.

Reply:
I think it is easiest if it is removed. If people want random filenames, there are ways to do this in shell scripts or Salt/Ansible or whatever they are using to orchestrate PATHspider. It's not PATHspider's job to do this so it's just more code to maintain.

Reply:
Agreed. In my use case I do the same.